test/test_ops.py

```diff
@@ -455,7 +455,7 @@ def _distance(a, b):
 
             # TODO: consider adding some tolerance to this comparison
             msg = f"Reference result was farther ({ref_distance}) from the precise " \
-                  "computation than the torch result was ({torch_distance})!"
+                  f"computation than the torch result was ({torch_distance})!"
```

Comment:
@ngimel and you said the automated static analysis wasn't helpful :^)

Reply:
nice fix